test/e2e/secrets_test.go

```diff
@@ -12,6 +12,7 @@ import (
 	"testing"
 	"time"
 
+	oconfig "github.com/openshift/api/config/v1"
```

Comment:
Like I mentioned in the [comment](https://github.com/openshift/windows-machine-config-operator/pull/508#discussion_r680095803), `oconfig` is not needed.